packages/serialization/json/src/jsonParseNode.ts

```diff
@@ -49,11 +52,14 @@ export class JsonParseNode implements ParseNode {
 	public getByteArrayValue(): ArrayBuffer | undefined {
 		const strValue = this.getStringValue();
 		if (strValue && strValue.length > 0) {
-			return inNodeEnv() ? Buffer.from(strValue, "base64").buffer : new TextEncoder().encode(strValue);
+			return Buffer.from(strValue, "base64").buffer;
```

Comment:
probably a regression during the merge here

Reply:
Ah yeah, good catch!